idemp-base/src/org/bandahealth/idempiere/base/model/MBandaSetup.java

```diff
@@ -1359,6 +1360,40 @@ public boolean configureClientUsers() {
 		});
 		return true;
 	}
+	
+	/**
+	 * Create default patients for new clients
+	 * @return
+	 */
+	public boolean createDefaultPatients() {
+		List<MBPartner_BH> patients = new Query(this.context, MBPartner_BH.Table_Name,
+				MBPartner_BH.COLUMNNAME_AD_Client_ID + "=?", getTransactionName()).setParameters(client.getAD_Client_ID())
```

Comment:
We'll need to pull this list from the config client, then copy each patient to the new client and save it. This will probably pull an empty list since no business partners should be created (unless they're the admin & user BPs).

Reply:
Good catch. That's what it was supposed to do. Not sure how I passed the client and not the config client id. Thanks